0.download_data/.jukit/.jukit_info.json

```diff
@@ -0,0 +1 @@
+{"cmd": "r\"\"\"\u00b0\u00b0\u00b0\n# Download sqlite plate data from aws\nNote, this script was not rerun to display the outputs for the sake of time. To download the data, you must be signed into your aws account\n\u00b0\u00b0\u00b0\"\"\"\n# |%%--%%| <TM56DRUlQQ|7T0gePh3Zn>\nr\"\"\"\u00b0\u00b0\u00b0\n## Imports\n\u00b0\u00b0\u00b0\"\"\"\n# |%%--%%| <7T0gePh3Zn|XqllLfeEBZ>\n\nimport subprocess\nimport pathlib\n\n# |%%--%%| <XqllLfeEBZ|q52bES5PJV>\nr\"\"\"\u00b0\u00b0\u00b0\n## Find the root of the git directory\nThis allows file paths to be referenced in a system agnostic way\n\u00b0\u00b0\u00b0\"\"\"\n# |%%--%%| <q52bES5PJV|xCbL8BuHjD>\n\n# Get the current working directory\ncwd = pathlib.Path.cwd()\n\nif (cwd / \".git\").is_dir():\n    root_dir = cwd\n\nelse:\n    root_dir = None\n    for parent in cwd.parents:\n        if (parent / \".git\").is_dir():\n            root_dir = parent\n            break\n\n# Check if a Git root directory was found\nif root_dir is None:\n    raise FileNotFoundError(\"No Git root directory found.\")\n\n# |%%--%%| <xCbL8BuHjD|DgPm3tAEsr>\nr\"\"\"\u00b0\u00b0\u00b0\n## Download the plate sqlite data from AWS S3\n\u00b0\u00b0\u00b0\"\"\"\n# |%%--%%| <DgPm3tAEsr|NDX1gPfGsN>\n\n# Specify the data path for downloading the data\ndownload_map = \"jump_paths.csv\"\n\n# Specify the location to save the data\nsave_location = f\"{root_dir}/big_drive/temp_plate_data\"\n\n# Download the data using a bash script\nsubprocess.run([\"bash\", \"download_from_aws.sh\", download_map, save_location])", "cmd_opts": " -s --md_cell_start=r\\\"\\\"\\\"\u00b0\u00b0\u00b0", "import_complete": 1, "terminal": "nvimterm"}
```

Comment:
I think this file (and other `.jukit` files) was added by mistake? Consider adding `.jukit` to gitignore.

Reply:
You're right, I thought I copied my previous .gitignore, thanks for catching this.